edge/test/integration/utils/helpers/helpers.go

```diff
@@ -83,6 +158,79 @@ func GetDeviceStateFromDB(deviceID string) string {
 	return device.State
 }
 
+func GetTwinAttributesFromDB(deviceID string, Name string) TwinAttribute {
+	var twinattribute TwinAttribute
+
+	pwd, err := os.Getwd()
+	if err != nil {
+		common.Failf("Failed to get PWD: %v", err)
+		os.Exit(1)
+	}
+	destpath := filepath.Join(pwd, "../../edge.db")
+	db, err := sql.Open("sqlite3", destpath)
+	if err != nil {
+		common.Failf("Open Sqlite DB failed : %v", err)
+	}
+	defer db.Close()
+	row, err := db.Query("SELECT * FROM device_twin")
+	defer row.Close()
+
+	for row.Next() {
+		err = row.Scan(&twinattribute.ID,
+			&twinattribute.DeviceID,
+			&twinattribute.Name,
+			&twinattribute.Description,
+			&twinattribute.Expected,
+			&twinattribute.Actual,
+			&twinattribute.ExpectedMeta,
+			&twinattribute.ActualMeta,
+			&twinattribute.ExpectedVer,
+			&twinattribute.ActualVer,
+			&twinattribute.Optional,
+			&twinattribute.Type,
+			&twinattribute.MetaData)
+
+		if err != nil {
+			common.Failf("Failed to scan DB rows: %v", err)
+		}
+		if string(twinattribute.DeviceID) == deviceID && twinattribute.Name == Name {
+			break
+		}
+	}
+
+	return twinattribute
+}
+
+func GetDeviceAttributesFromDB(deviceID string, Name string) Attribute {
+	var attribute Attribute
+
+	pwd, err := os.Getwd()
+	if err != nil {
+		common.Failf("Failed to get PWD: %v", err)
+		os.Exit(1)
+	}
+	destpath := filepath.Join(pwd, "../../edge.db")
+	db, err := sql.Open("sqlite3", destpath)
+	if err != nil {
+		common.Failf("Open Sqlite DB failed : %v", err)
+	}
+	defer db.Close()
+	row, err := db.Query("SELECT * FROM device_attr")
+	defer row.Close()
+
+	for row.Next() {
+		err = row.Scan(&attribute.ID, &attribute.DeviceID, &attribute.Name, &attribute.Description, &attribute.Value, &attribute.Optional, &attribute.Type, &attribute.MetaData)
+		if err != nil {
+			common.Failf("Failed to scan DB rows: %v", err)
+		}
+		if string(attribute.DeviceID) == deviceID && attribute.Name == Name {
+			break
+		}
+	}
+
```

Comment:
remove extra new lines throughout the code.

Reply:
I have added some new lines between two functions, to make it look better and segregated.